nuxt.config.js

```diff
@@ -0,0 +1,6 @@
+import config from '@rancher/shell/nuxt.config';
```

Comment:
As far as I can tell, this file is for when we are developing a plugin. Maybe it would be good to add a comment to it saying this? I can see someone who isn't familiar with Nuxt getting confused over why we have both nuxt.config.js and src/nuxt.config.js.

Reply:
I think this is a good idea. I added a short comment on the purpose of this config. Let me know what you think.